runtime/lua/vim/lsp.lua

```diff
@@ -687,12 +687,12 @@ function lsp.start_client(config)
   ---
   ---@param method (string) LSP method name
   ---@param params (table) The parameters for that method
-  function dispatch.server_request(method, params)
-    local _ = log.debug() and log.debug('server_request', method, params)
+  function dispatch.server_request(method, result)
```

Comment:
docstring outdated

Reply:
I changed it back, to params, I think result make sense in the signature of our handlers, but we should stick with params in the dispatch function table, as that maps directly onto the spec (my bad for changing it originally in the PR, see my above comment earlier in the thread.